hs_core/models.py

```diff
@@ -418,25 +418,30 @@ def page_permissions_page_processor(request, page):
     if is_owner or (cm.raccess.shareable and (is_view or is_edit)):
         show_manage_access = True
 
-    max_file_size = getattr(settings, 'FILE_UPLOAD_MAX_SIZE', 1024 * 25)
+    max_file_size = getattr(settings, 'FILE_UPLOAD_MAX_SIZE', 25 * 1024**3)
     remaining_quota = get_remaining_user_quota(cm.quota_holder, "MB")
     if remaining_quota is not None:
-        max_file_size = min(max_file_size, remaining_quota)
```

Comment:
yes, and the min  between remaining_quota and max_file_size was removed.

Reply:
I think it should effectively be replaced by:
https://github.com/hydroshare/hydroshare/pull/5674/files/f1adc3f7a935c6bfaf05bd2caa31ebeafea46dd4#diff-7f6bf4eb5e5d92ef296bf7690355fa78a036899688ef91e6e35e7394f3628aacR26

Reply:
You're right, that should work as a replacement.

Reply:
and I just checked to make sure it's enforced on the backend as well:
https://github.com/hydroshare/hydroshare/blob/rc-2.15.10/hs_core/hydroshare/resource.py#L333